Sites/GD2/Project 1/Styles.css

```
@@ -0,0 +1,105 @@
+img {
+    max-width: 100%
+}
+
+h1 {
+    color: #260648;
+    font-weight: bold;
+    font-size: 40px
+}
+
+bodycolor {
```

If what you want is all the text, unless overridden by a different rule, to be color #F6F6EC, this is what you want to update the selector to:
```suggestion
body {
```